README.md

```diff
@@ -347,6 +347,17 @@ li {
     </tr>
 </table>
 
+## Getting Help
+
+Please, do not open issues for general support questions as we want to keep GitHub issues for bug reports and feature requests. You've got much better chances of getting your question answered on [StackOverflow](http://stackoverflow.com/questions/tagged/ng-bootstrap) where maintainers are looking at questions tagged with `loryJS`.
+
+StackOverflow is a much better place to ask questions since:
+* There are hundreds of people willing to help on StackOverflow
+* Questions and answers stay available for public viewing so your question / answer might help someone else
+* SO voting system assures that the best answers are prominently visible.
```

Comment:
Or keep it and change `SO voting system assures that the best answers are prominently visible.` to `The Stack Overflow voting system assures that the best answers are prominently visible.`

I needed a minute to understand what you meant with `SO` thats the reason for the change.